vos/vos/vos.py

```diff
@@ -1618,15 +1621,16 @@ def copy(self, source, destination, send_md5=False, disposition=False):
                         return destination_size
```

Comment:
I think in the above 'else' statement you should put ```elseif not fhead:``` since that would then prevent this md5 check from happening when an fhead is being requested.  Also, that would prevent check_md5 from being set to True and remove the need for the if statement in the lines below.